zk/stages/stages.go

```diff
@@ -58,6 +55,8 @@ func DefaultZkSequencerStages(
 				return stagedsync.PruneBlockHashStage(p, tx, blockHashCfg, ctx)
 			},
 		},
```

Comment:
If we're the only sequencer in operation would this not be covered during execution/sequencing?

Reply:
Not clear in the UI, but this is targetted at the block hashes stage

Reply:
yeah, yeah, this will be during the execution, good catch

Reply:
removed that stage